galaxy_mobile/lib/models/main_store.dart

```diff
@@ -106,7 +106,7 @@ class MainStore extends ChangeNotifier {
     if (config == null) {
       config = await _api.fetchConfig();
     }
-    if( spec == null) {
+    if (spec == null) {
```

Comment:
why did you remove this condition?

Reply:
Which condition? I didn't remove anything just modified spacing